minutes/models/base/cnn/params.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "cnn",
+    "ms_per_observation": 3000,
```

Comment:
3 seconds is what I got he best accuracy on (sans GPU).

Reply:
Well, it makes sense, you have bigger observation window, but it might not be suitable for all tasks :( do you remember what was the accuracy for 1-second samples?

Reply:
Theres quite a bit of background to cover here.

1. I removed the `matplotlib` spectrograms, `scipy.signal.spectrogram` is faster and produces flat spectrogram arrays with no color channels. See [this PR](https://github.com/ubclaunchpad/minutes/pull/107) for more details.
2. New spectrogram shape became `(129, 196)`, so previous _models_ built on GPU's no longer predict on new data (they had a shape like `(1024, 32, 3)` or something like that).
3. Previous model _code_ no longer trained as well because architectures included layers that expected 3 dimensional inputs ([I reproduced our model for flat spectrograms here](https://github.com/ubclaunchpad/minutes/blob/master/minutes/base.py#L78-L86)). 
4. After achieving ~95% accuracy on a transfer model using a CPU with the above decisions made, I felt like with small changes to model architecture and additional training data, we could improve on my [base model](https://github.com/ubclaunchpad/minutes/blob/master/minutes/models/cnn.h5) iteratively (please feel free to open a PR!).
5. As you can imagine, with the above changes, many more things affect accuracy than just `ms_per_observation` (ie, by dropping the `ms_per_observation`, you would not magically improve this model). On the contrary, increasing `ms_per_observation` helped this particular model—and yes it assumes people have enough training data so that many 3s segments can be created, which is not great!
6. Theres no longer 1 model to rule them all, users can define their own base models (and we should define new ones and version control them here). [Here is an example](https://github.com/ubclaunchpad/minutes/issues/125) of how you do that.

Reply:
Oh ok, I see, if we can define our own models and specify ```ms_per_observation``` by ourselves, which is good. 
My concern was just that for this particular base model, for some conversation scenarios, 3s might be too if speakers switch, say, in the middle of a few observations. This may lead to assigning a significant portion of one speaker's speech to another speaker. 
But this is a difficult tradeoff, we drop ```ms_per_observation``` and get less accuracy, or we increase ```ms_per_observation``` and make the model less "agile" when dealing with scenarios above.

Reply:
Unfortunately, we can probably only measure this tradeoff if we add transcription, I guess.